v2/pkg/templates/workflows.go

```diff
@@ -71,10 +74,19 @@ func parseWorkflowTemplate(workflow *workflows.WorkflowTemplate, preprocessor Pr
 			gologger.Warning().Msgf("Could not parse workflow template %s: no executer found\n", path)
 			continue
 		}
+		workflowTemplates = append(workflowTemplates, template)
+	}
+
+	finalTemplates, clusterCount := ClusterTemplates(workflowTemplates, options.Copy())
+	if clusterCount > 0 {
+		gologger.Info().Msgf("Templates clustered at same workflow level: %d ", clusterCount)
```

Comment:
We should not display this info at each workflow level. We can omit this logging line here

Reply:
done